src/js/translate-visitor.ts

```diff
@@ -7,15 +7,17 @@ const n = types.namedTypes;
 const b = types.builders;
 const TRANSLATE_SERVICE_NAME = "$translate";
 
-export default class TranslateVisitor extends types.PathVisitor implements  types.Visitor {
+export default class TranslateVisitor extends types.PathVisitor implements types.Visitor {
     changedAst: boolean = false;
     comments: acorn.Comment[]= [];
     tokens: acorn.Token[] = [];
     options: acorn.Options = {
         locations: true,
         onComment: this.comments,
         onToken: this.tokens,
-        ranges: true
+        ranges: true,
+        sourceType: "module",
```

Comment:
I don't think we should enable this by default. The problem is that this enables strict mode for all files. So, this might fail to parse in case someone is still using commonjs with any feature that is not supported by ES6.

So I think it would be best if we allow the user to pass options to the js-loader (like you can to other loaders) and merge them with the default options defined by the webpack-plugin....